lib/batch_message.js

```diff
@@ -0,0 +1,26 @@
+function Batch () {
+	var messages = Array.prototype.slice.call(arguments)
+
+	this.op = 'batch'
+	this.value = messages
+}
+
+Object.defineProperty(Batch.prototype, 'length', {
```

Comment:
yes, IE9 (the lowest common denominator) supports Object.defineProperty http://kangax.github.io/compat-table/es5/#ie9